### PR TITLE
Produce geometries 

### PR DESCRIPTION
**Pull request by tankedman**
Refactors the get_links_between_two_nodes function to output geometries - creating shell function and a query function.

### CODE REVIEW

**Comment by Nate-Wessel** [comment]
@tankedman is this ready for review? It still says draft.

File: backend/app/routes.py [hunk 1]
```diff
@@ -101,9 +103,30 @@ def get_links_between_two_nodes(from_node_id, to_node_id):
...
+        "geometry": {
+            "type": "MultiLineString",
+            "coordinates": [ link['geometry']['coordinates'] for link in links ]
+        }
+    }
+    return jsonify(shortest_link_data)
+
+
+#Function that returns a json with geometries of links between two nodes
+@app.route('/get-link-geom/<from_node_id>/<to_node_id>', methods=['GET'])
```
Comment (Nate-Wessel) [inline backend/app/routes.py]: Why expose this function? It's only used internally, no?

File: backend/app/routes.py [hunk 2]
```diff
@@ -101,9 +103,30 @@ def get_links_between_two_nodes(from_node_id, to_node_id):
...
+def get_link_geom(from_node_id, to_node_id):
```
Comment (Nate-Wessel) [inline backend/app/routes.py]: I wonder if `get_links` would be a clearer name for this? It's returning a set of links, not a geom.

File: backend/app/routes.py [hunk 3]
```diff
@@ -101,9 +103,30 @@ def get_links_between_two_nodes(from_node_id, to_node_id):
...
+        "target": to_node_id,
+        "links": links,
+        # the following three fields are for compatibility and should eventually be removed
+        "path_name": "",
+        "link_dirs": [ link['link_dir'] for link in links ],
+        "geometry": {
+            "type": "MultiLineString",
+            "coordinates": [ link['geometry']['coordinates'] for link in links ]
+        }
+    }
```
Comment (Nate-Wessel) [inline backend/app/routes.py:118]: Move this to within the `get_links` function

Comment (Nate-Wessel) [reply backend/app/routes.py:118]: WAIT

Comment (Nate-Wessel) [reply backend/app/routes.py:118]: No, actually, I'm reading my comment on line 114 and realizing we should just leave this as is. This is parsing the link lines into a `MultiLineString` (similar to the `GeometryCollection` I mentioned) which will make it harder to associate individual links to their respective line geometries. I was only doing that here for compatibility with the frontend which will change eventually. Sorry for the confusion.

File: backend/app/routes.py (hunk 1 above)
Comment (tankedman) [reply backend/app/routes.py]: endpoint removed

File: backend/app/routes.py (hunk 2 above)
Comment (tankedman) [reply backend/app/routes.py]: changed

**Pull request merged**